controller/linodemachine_controller.go

```diff
@@ -103,76 +103,31 @@ func (r *LinodeMachineReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	linodeMachine := &infrav1alpha1.LinodeMachine{}
 	if err := r.Client.Get(ctx, req.NamespacedName, linodeMachine); err != nil {
 		if err = client.IgnoreNotFound(err); err != nil {
-			log.Error(err, "Failed to fetch Linode machine")
+			log.Error(err, "Failed to fetch LinodeMachine")
 		}
 
 		return ctrl.Result{}, err
 	}
 
-	machine, err := kutil.GetOwnerMachine(ctx, r.Client, linodeMachine.ObjectMeta)
-	switch {
-	case err != nil:
-		if err = client.IgnoreNotFound(err); err != nil {
-			log.Error(err, "Failed to fetch owner machine")
-		}
-
+	machine, err := r.getOwnerMachine(ctx, *linodeMachine, log)
+	if err != nil || machine == nil {
 		return ctrl.Result{}, err
-	case machine == nil:
-		log.Info("Machine Controller has not yet set OwnerRef, skipping reconciliation")
-
-		return ctrl.Result{}, nil
-	case skippedMachinePhases[machine.Status.Phase]:
-
-		return ctrl.Result{}, nil
-	default:
-		match := false
-		for i := range linodeMachine.OwnerReferences {
-			if match = linodeMachine.OwnerReferences[i].UID == machine.UID; match {
-				break
-			}
-		}
-
-		if !match {
-			log.Info("Failed to find the referenced owner machine, skipping reconciliation", "references", linodeMachine.OwnerReferences, "machine", machine.ObjectMeta)
-
-			return ctrl.Result{}, nil
-		}
 	}
+	log = log.WithValues("LinodeMachine: ", machine.Name)
```

Comment:
The `WithValues` method takes a key and value pair. I don't think you need to specify the `: ` separator in the key argument.